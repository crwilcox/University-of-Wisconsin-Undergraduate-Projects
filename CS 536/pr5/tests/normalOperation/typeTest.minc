int main() {
	int x;
	double y;
	bool z;
	x = 1 + 1;
	x = 4 - 8;
	y = 2.0 + 2.0;
	y = 3.5 - 2.32;
	y = 4.6 * 5.4e2;
	z = true;
	z = true || false;
	z = true && true;
	return x;
}
