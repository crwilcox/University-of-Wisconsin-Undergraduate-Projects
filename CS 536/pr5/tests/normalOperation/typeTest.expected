int main() {
  int x;
  double y;
  bool z;
  (x = (1 + 1)); /* int */
  (x = (4 - 8)); /* int */
  (y = (2.0 + 2.0)); /* double */
  (y = (3.5 - 2.32)); /* double */
  (y = (4.6 * 540.0)); /* double */
  (z = true); /* bool */
  (z = (true || false)); /* bool */
  (z = (true && true)); /* bool */
  return x;
}
